handlers/challenge_handler.py

```diff
@@ -500,7 +508,10 @@ def init(self, slack_wrapper):
                     ctf.add_challenge(challenge)
 
         # Create the database accordingly
-        pickle.dump(database, open(self.DB, "wb+"))
+        pickle.dump(database, open(ChallengeHandler.DB, "wb+"))
+
+    def init(self, slack_wrapper):
+        ChallengeHandler.update_database_from_slack(slack_wrapper)
 
 # Register this handler
 HandlerFactory.register("ctf", ChallengeHandler())
```

Comment:
See that the challenge handler is instantiated here. We probably don't need to if we are keeping all state in the class object itself.

Reply:
I just made this "static", so we can call it from the ReloadCommand, which has no reference to the challenge handler.

Reply:
Yeah I filed #86 to figure out how we want to do this. This is fine for now.